addons/hr_expense/models/hr_expense.py

```diff
@@ -387,42 +424,68 @@ def action_view_sheet(self):
         }
 
     def _get_default_expense_sheet_values(self):
-        if any(expense.state != 'draft' or expense.sheet_id for expense in self):
+        # If there is an expense with total_amount_company == 0, it means that expense has not been processed by OCR yet
+        expenses_with_amount = self.filtered(lambda expense: not float_compare(expense.total_amount_company, 0.0, precision_rounding=expense.company_currency_id.rounding) == 0)
+
+        if any(expense.state != 'draft' or expense.sheet_id for expense in expenses_with_amount):
             raise UserError(_("You cannot report twice the same line!"))
-        if len(self.mapped('employee_id')) != 1:
+        if not expenses_with_amount:
+            raise UserError(_("You cannot report the expenses without amount!"))
+        if len(expenses_with_amount.mapped('employee_id')) != 1:
             raise UserError(_("You cannot report expenses for different employees in the same report."))
-        if any(not expense.product_id for expense in self):
+        if any(not expense.product_id for expense in expenses_with_amount):
             raise UserError(_("You can not create report without category."))
 
-        todo = self.filtered(lambda x: x.payment_mode=='own_account') or self.filtered(lambda x: x.payment_mode=='company_account')
-        if len(todo) == 1:
-            expense_name = todo.name
-        else:
-            dates = todo.mapped('date')
-            min_date = format_date(self.env, min(dates))
-            max_date = format_date(self.env, max(dates))
-            expense_name = min_date if max_date == min_date else "%s - %s" % (min_date, max_date)
-
-        values = {
-            'default_company_id': self.company_id.id,
-            'default_employee_id': self[0].employee_id.id,
-            'default_name': expense_name,
-            'default_expense_line_ids': [Command.set(todo.ids)],
-            'default_state': 'draft',
-            'create': False
-        }
+        # Check if two reports should be created
+        own_expenses = expenses_with_amount.filtered(lambda x: x.payment_mode == 'own_account')
+        company_expenses = expenses_with_amount.filtered(lambda x: x.payment_mode == 'company_account')
+        create_two_reports = bool(own_expenses) and bool(company_expenses)
```

Comment:
The bool() is not required here
```suggestion
        create_two_reports = own_expenses and company_expenses
```

Reply:
done :)